meta/state.go

```diff
@@ -209,11 +209,6 @@ func (r *localRaft) close() error {
 		r.transport = nil
 	}
 
-	if r.raftLayer != nil {
```

Comment:
Can you add a comment to the block above, stating that that `Close()` will close the `raftLayer()`?